README.md

```diff
@@ -112,5 +112,6 @@ This software is released under the terms of the MIT License. See the
 
 ## Find us
 
-Jangouts developers can be usually found at
-[#jangouts IRC channel at Freenode](irc://irc.freenode.net/jangouts)
+Jangouts developers can be usually found at:
+ - [#jangouts IRC channel at Freenode](https://webchat.freenode.net/?channels=%23jangouts)
+ - [Google Groups](https://groups.google.com/forum/#!forum/jangouts)
```

Comment:
Well, if we are going to look that close, :wink: I would even state that IRC is good to reach the developers at a given point in time and the Google group is more targeted for general discussions within the whole community. However you want to word that.